python/python/lance/dataset.py

```diff
@@ -1105,7 +1117,12 @@ def cleanup_old_versions(
     def create_scalar_index(
         self,
         column: str,
-        index_type: Union[Literal["BTREE"], Literal["BITMAP"], Literal["LABEL_LIST"]],
+        index_type: Union[
```

Comment:
Can you document the `INVERTED` index?  E.g. somewhere around the paragraph starting with `There are three types of scalar indices available today`...